pandas/io/excel/_openpyxl.py

```diff
@@ -61,17 +61,23 @@ def __init__(
             self.sheets = {name: self.book[name] for name in self.book.sheetnames}
 
         else:
-            # Create workbook object with default optimized_write=True.
-            self.book = Workbook()
+            try:
+                # Sheets are not automatically created in the workbook
+                self.book = Workbook(write_only=True)
+            except ImportError:
```

Comment:
If you want to know if `lxml` is installed, use `import lxml`. This `ImportError` can be generated by different missing dependencies.

Reply:
> If you want to know if `lxml` is installed, use `import lxml`. This `ImportError` can be generated by different missing dependencies.

Sounds good, I'll make that change now.